src/components/InteractiveMap/InteractiveMap.vue

```diff
@@ -54,49 +36,14 @@
       </v-container>
       <v-divider></v-divider>
       <v-container>
-        <v-card v-for="card in cards" :key="JSON.stringify(card)">
-          <v-toolbar dense color="primary" dark>
-            <v-toolbar-title class="body-2">{{ card.name }}</v-toolbar-title>
-            <v-spacer></v-spacer>
-            <v-icon>close</v-icon>
-          </v-toolbar>
-          <v-card-title primary-title class="py-2">
-            <v-layout justify-space-around>
-              <v-btn flat icon>
-                <v-icon>info</v-icon>
-              </v-btn>
-              <v-btn flat icon>
-                <v-icon>edit</v-icon>
-              </v-btn>
-            </v-layout>
-            <v-layout wrap>
-              <v-flex class="xs6">
-                Organism:
-              </v-flex>
-              <v-flex class="xs6 text-xs-right">
-                {{ card.organism.name }}
-              </v-flex>
-              <v-flex class="xs6">
-                Model:
-              </v-flex>
-              <v-flex class="xs6 text-xs-right">
-                {{ card.model.name }}
-              </v-flex>
-              <v-flex class="xs6">
-                Method:
-              </v-flex>
-              <v-flex class="xs6 text-xs-right">
-                {{ card.method }}
-              </v-flex>
-              <v-flex class="xs6">
-                Growth rate:
-              </v-flex>
-              <v-flex class="xs6 text-xs-right">
-                {{ card.growthRate }} <em>h<sup>-1</sup></em>
-              </v-flex>
-            </v-layout>
-          </v-card-title>
-        </v-card>
+        <Card
+          v-for="card in cards"
+          :key="cards.indexOf(card)"
```

Comment:
Two issues here:

1. Finding the index of the card is unnecessary, you can get the index from the [v-for](https://vuejs.org/v2/guide/list.html)
```js
v-for="(item, index) in items"
```
2. I'm not 100% certain on this but I think using the index as key can lead to problems with components not being destroyed properly. Say you create new cards that then end up at the same index. I think this can be messy. I'll need to read up on that, though.

Reply:
Yes, this was intended temporarily. I'm not sure what to use for key, maybe we need to generate uuids for the cards since they don't necessarily always have a design primary key. I'll add a comment about this